<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="DESKTOP-888OH8L" name="tests.Orderproducts" tests="4" failures="0" timestamp="13 Jul 2020 05:40:45 GMT" time="60.379" errors="1">
  <testcase name="orderProducts" time="0.175" classname="tests.Orderproducts">
    <skipped/>
  </testcase> <!-- orderProducts -->
  <testcase name="orderProducts" time="58.511" classname="tests.Orderproducts">
    <error type="java.lang.Exception">
      <![CDATA[java.lang.Exception
at util.TestUtil.Sendkeys(TestUtil.java:59)
at pages.SignUpPage.createAcc(SignUpPage.java:49)
at tests.Orderproducts.orderProducts(Orderproducts.java:93)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- orderProducts -->
  <testcase name="orderProducts" time="0.371" classname="tests.Orderproducts">
    <skipped/>
  </testcase> <!-- orderProducts -->
  <testcase name="orderProducts" time="1.322" classname="tests.Orderproducts">
    <skipped/>
  </testcase> <!-- orderProducts -->
</testsuite> <!-- tests.Orderproducts -->
